DRIVERS/MDIS_LL/SMB2/TOOLS/SMB2_SHC_CTRL/COM/smb2_shc_ctrl.c

```diff
@@ -394,6 +427,78 @@ static void set_temperature(int32 tempC)
 	}
 }
 
+/****************************************************************************/
+/** Print status of persistent power button
+*/
+static void print_persistent_pwrbtn_status()
+{
+	struct shc_fwversion firm_version;
+        int err = SMB2SHC_GetFirm_Ver(&firm_version);
+	if (err) {
+		PrintError("***ERROR: SMB2SHC_GetFirm_Ver:", err);
+                return err;
+	}
+        if (firm_version.min_revision <= 16) {
```

Comment:
Missing compare of maj_revision.
E.g. rev. 5.1 would lead to "button is not supported"